docs/src/use/configure/configuration-files.md

```diff
@@ -495,3 +498,152 @@ npx eslint --config some-other-file.js **/*.js
 ```
 
 In this case, ESLint does not search for `eslint.config.js` and instead uses `some-other-file.js`.
+
+<a name="experimental-typescript-support">
+
+## ◆ Configuration Files written in TypeScript (new)
+
+</a>
+
+As of ESLint v9.x, you can write your ESLint configuration files in [TypeScript](https://www.typescriptlang.org).
+
+::: warning
+This feature is currently experimental and may change in future versions.
+:::
+
+To use this feature, you must have [`jiti`](https://github.com/unjs/jiti) installed in your project.
+
+NPM
+
+```bash
+npm install --save-dev jiti
+```
+
+Yarn
+
+```bash
+yarn add --dev jiti
+```
+
+PNPM
+
+```bash
+pnpm add --save-dev jiti
+```
```

Comment:
We can tighten this up.
```suggestion

````bash
npm install jiti -D
# or
yarn add --dev jiti
# or
pnpm add jiti -D
````
```